Apps/Laboratory/src/Controllers/LaboratoryController.cs

```diff
@@ -78,11 +78,33 @@ public async Task<RequestResult<IEnumerable<Covid19Order>>> GetCovid19Orders([Fr
             return result;
         }
 
+        /// <summary>
+        /// Gets a json summary of laboratory summary.
+        /// </summary>
+        /// <param name="hdid">The hdid resource to request the Laboratory summary for.</param>
+        /// <returns>A summary of Laboratory records wrapped in a request result.</returns>
+        /// <response code="200">Returns a Laboratory Summary containing a list of Laboratory orders.</response>
+        /// <response code="401">The client must authenticate itself to get the requested response.</response>
+        /// <response code="403">The client does not have access rights to the content; that is, it is unauthorized, so the server is refusing to give the requested resource. Unlike 401, the client's identity is known to the server.</response>
+        /// <response code="503">The service is unavailable for use.</response>
+        [HttpGet]
+        [Produces("application/json")]
+        [Route("LaboratorySummary")]
+        [Authorize(Policy = LaboratoryPolicy.Read)]
+        public async Task<RequestResult<LaboratorySummary>> GetLaboratorySummary([FromQuery] string hdid)
+        {
+            this.logger.LogDebug($"Getting laboratory summary... ");
+            RequestResult<LaboratorySummary> result = await this.service.GetLaboratorySummary(hdid).ConfigureAwait(true);
+            this.logger.LogDebug("Finished getting laboratory summary from controller for Hdid: {Hdid}...", hdid);
+            return result;
+        }
```

Comment:
I think we should return `RequestResult<IEnumerable<LaboratoryOrder>>` since we don't need the extra properties from `LaboratorySummary`.

Reply:
Returning an Enumerable of LaboratoryOrders now.